lite2/client.go

```diff
@@ -870,6 +870,10 @@ func (c *Client) backwards(trustedHeader *types.SignedHeader, newHeader *types.S
 		trustedHeader = interimHeader
```

Comment:
do you think it's possible to get rid of `trustedHeader` variable and just use `interimHeader`?

Reply:
you need at least two headers that change incrementally towards the target newHeader so I don't think that is possible